### DISCUSSION TITLE
Getting RSAException  Unhandled Exception: Instance of 'RSAException'

### DISCUSSION THREAD

**Discussion by Manjunath0987** [jerson/rsa-mobile#10] [Q&A]
I am using 
fast_rsa version 3.4.6
in flutter we are using above plugin for 
RSA PKCS#1 encryption and OAEP encryption and go the below exception 

E/flutter (32269): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: Instance of 'RSAException'
E/flutter (32269): #0      RSA._stringResponse (package:fast_rsa/fast_rsa.dart:61:7)
E/flutter (32269): <asynchronous suspension>
E/flutter (32269): #1      RSA.encryptPKCS1v15 (package:fast_rsa/fast_rsa.dart:303:12)
E/flutter (32269): <asynchronous suspension>
E/flutter (32269): #2      WelcomeWidget.initializeApiReq (package:fastag/src/view/ui/widgets/onboarding/welcome_widget.dart:307:22)
E/flutter (32269): <asynchronous suspension>
E/flutter (32269): 

flutter version is:
[✓] Flutter (Channel stable, 2.10.0, on macOS 12.6 21G115 darwin-x64, locale en-GB)
[!] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    ✗ cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.
[✓] Xcode - develop for iOS and macOS (Xcode 13.4.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] Connected device (2 available)
[✓] HTTP Host Availability

! Doctor found issues in 1 category.

Any idea on this issue

method calling is:
var encryptHex = await RSA.encryptPKCS1v15(randomKey, publickey fetching from pem file)

**Comment by jerson** [comment]
Hi, could be related to the public key, please add an example with a sample public key to try to reproduce it here
